tasties_app/views.py

```diff
@@ -70,5 +71,20 @@ def view_recipe(request, recipe=None):
     ingredients = recipe.ingredient_set.all()
     rating = recipe.rating_set.aggregate(Avg('rating'))['rating__avg']
     categories = recipe.categories.all()
-    context = {'recipe': recipe, 'ingredients': ingredients, 'rating': rating, 'categories': categories}
+    if request.method == "POST" and request.POST['action'] == 'Comment':
+        add_comment(request, recipe.title)
```

Comment:
Would passing the actual `Recipe` object instead of its title be a worse idea? In any case, even though our `Recipe` objects have unique `title`s, I think it would be best practice to pass their PK (the field recipe.id) instead.

Reply:
I change it to like you suggest